webhook_to_fedora_messaging/models/apikey.py

```diff
@@ -0,0 +1,23 @@
+# SPDX-FileCopyrightText: Contributors to the Fedora Project
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from sqlalchemy import Column, Integer, UnicodeText, Boolean, ForeignKey
+from webhook_to_fedora_messaging.models.util import UUIDCreatableMixin, CreatableMixin, TZDateTime
+
+from webhook_to_fedora_messaging.database import Base
+
+from uuid import uuid4
+
+
+class APIKey(Base, UUIDCreatableMixin, CreatableMixin):
+    __tablename__ = "apikeys"
+
+    id = Column("id", Integer, primary_key=True, nullable=False)
+    user_id = Column(
+        "user_id", Integer, ForeignKey("user.id", ondelete="CASCADE"), unique=False, nullable=False
+    )
+    name = Column("name", UnicodeText, nullable=False)
+    hash = Column("hash", UnicodeText, unique=True, nullable=False, default=uuid4().hex[0:8])
```

Comment:
`hash` is a global function in python, let's just use `value` or something generic. Besides, it's not even a hash :-)
Also, instead of keeping only the first 8 chars of UUID, you can use [secrets.token_urlsafe](https://docs.python.org/3/library/secrets.html#secrets.token_urlsafe).

Reply:
How about the name `token` and the value being just `uuid4().hex`?